notebooks/18.In_Depth-Trees_and_Forests.ipynb

```diff
@@ -100,8 +100,8 @@
     "y_fit_1 = reg.predict(X_fit)\n",
     "\n",
     "plt.figure()\n",
-    "plt.plot(X_fit.ravel(), y_fit_1, color='blue', label=\"prediction\")\n",
-    "plt.plot(X.ravel(), y, '.k', label=\"training data\")\n",
+    "plt.plot(X_fit.ravel(), y_fit_1, color='tab:blue', label=\"prediction\")\n",
+    "plt.plot(X.ravel(), y, 'C7.', label=\"training data\")\n",
```

Comment:
what's that?

Reply:
Grey from the tab10

Reply:
I use C because it did not work with the dot next

Reply:
I don't know this syntax though...